### DISCUSSION TITLE
Using web3 domain names as wallet addresses.

### DISCUSSION THREAD

**Discussion by Mattweb3** [phantom/docs#132] [General]
![Screenshot_20231208-095624_Gmail](https://github.com/phantom/docs/assets/153277483/9fd63369-ae1f-48c7-909f-2d2f21d39970)
Hi, 

So I reached out to support and they asked me to come and ask here. 

I own multiple TLDs that are currently on the Handshake blockchain. The devs working on Handshake managed to bridge TLDs to a register on Ethereum that allows people to mint subdomains of the TLD. The subdomains are on Ethereum and can now be used as wallet addresses for Metamask as they work very similar to ENS. 

One of the many TLDs I own is .wagbo. I want to be able to use .wagbo names as wallet addresses for phantom. The .wagbo name is currently still on the Handshake blockchain. I can bridge this to Ethereum via impervious domain register or I could keep it on Handshake and use a register there. Is there a way that we can make the names usable with Phantom?  I know there are ways to bridge individual NFTs from Ethereum to Solana but I doubt that would make the names usable as wallet addresses. Metamask had to add something their side to accept the names The subdomains can also be converted to ENS via the impervious website. 

To recap, I want to be able to use Handshake names as Phantom wallet addresses. Can we make it happen?

**Comment by adamdelphantom** [comment]
Hi @Mattweb3,

Thank you for writing in with this. I will create an internal ticket for this as a feature request. If you haven't already I would recommend creating a feature request on our [canny board](https://phantom.canny.io/admin/feedback/feature-requests/p/portfolio-watchlist-views) and asking anyone that would like to see this ability to vote on the feature request. Our product team is constantly reviewing our canny feature board for prioritizing features.
